projects/worker-metadata/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "example-worker-metadata",
+    "author": "Adobe Inc.",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "description": "Asset Compute example custom metadata worker",
+    "repository": "adobe/example-worker-metadata",
+    "dependencies": {
+        "@adobe/asset-compute-sdk": "^2.2.1",
+        "@adobe/asset-compute-xmp": "^1.0.0"
+    },
+    "scripts": {
+        "test": "aio asset-compute test-worker"
+    },
+    "keywords": [
+        "asset compute",
+        "adobe",
+        "serverless",
+        "example",
+        "worker"
```

Comment:
+ metadata
+ xmp 
(So it will appear in search results for metadata and xmp on npm's website).

Reply:
^^^ keywords are not really necessary unless you plan to release this as a library to npm

Reply:
yes, and at this time I don't see actions/workers ever be released. there is zero tooling to use an npm published worker from `aio` in any way (unless it doubles as a library but that doesn't make sense). although if we ask them to support what we do with `nui` and nui/app it would go that way, via private registries.

but for now these publication related fields can be removed from examples and instead put in `"private": true`.

Reply:
I've applied that change